components/dropbox/sources/all-updates/all-updates.mjs

```diff
@@ -1,25 +1,35 @@
-const dropbox = require("../../dropbox.app.js");
+import dropbox from "../../dropbox.app.mjs";
```

Comment:
Since all 3 sources share some things in common, I'd recommend creating a `common.mjs` file in the sources directory. It would include 
props: `dropbox`, `db`, `path`, `recursive`, & `dropboxApphook`
method: `getMeta()`
and
```
  hooks: {
    async activate() {
      await this.dropbox.initState(this);
    },
  }, 
```